src/components/alert/SpshAlert.vue

```diff
@@ -0,0 +1,75 @@
+<script setup lang="ts">
+  defineProps<{
+    modelValue: boolean
+    title: string
+    text: string
+    showButton: boolean
+    buttonText?: string
+    buttonClass?: string
+    buttonAction?: Function
+    type: 'error' | 'success' | 'warning' | 'info' | undefined
+    closable?: boolean
+  }>()
+
+  type Emits = {
+    (event: 'update:modelValue', value: boolean): void
+  }
+  const emit: Emits = defineEmits<{
+    (event: 'update:modelValue', value: boolean): void
+  }>()
+
+  const closeAlert = (): void => {
+    emit('update:modelValue', false) // Emit the event to update modelValue
+  }
+</script>
+
+<template>
+  <v-container>
+    <v-slide-y-transition>
+      <v-alert
+        v-if="modelValue"
```

Comment:
Mit `v-if=...` baust du die Funktionalität des Alerts nach.
```suggestion
        :modelValue="modelValue"
```

Reply:
Done